src/HaskellWorks/Data/Json/Standard/Cursor/Internal/IbBp.hs

```diff
@@ -13,12 +13,9 @@ data IbBp = IbBp
   , bp :: DVS.Vector Word64
   }
 
-class ToIbBp a where
-  toIbBp :: a -> IbBp
-
-instance ToIbBp BS.ByteString where
-  toIbBp bs = IbBp
-    { ib = J.toInterestBits64 blankedJson
-    , bp = J.toBalancedParens64 blankedJson
-    }
-    where blankedJson = J.toBlankedJsonTyped bs
+slowToIbBp :: BS.ByteString -> IbBp
+slowToIbBp bs = IbBp
+  { ib = J.toInterestBits64 blankedJson
+  , bp = J.toBalancedParens64 blankedJson
+  }
+  where blankedJson = J.toBlankedJsonTyped bs
```

Comment:
This code will actually load a lot of intermediate data into memory, which is bad.  This is because `blankedJson` is a lazy list that is used to build to vectors one after another.  If it was used to build a single vector this would stream and wouldn't be a problem, but building to simultaneously is problematic.